src/api/app/queries/autocomplete_finder_base.rb

```diff
@@ -0,0 +1,7 @@
+class AutocompleteFinderBase
```

Comment:
We're starting to have more and more query objects, I think we could reorganize those `AutocompleteFinder` classes.

How about having the classes like this:
- src/api/app/queries/autocomplete_finder.rb
- src/api/app/queries/autocomplete_finder/package.rb
- src/api/app/queries/autocomplete_finder/project.rb

The parent class (`src/api/app/queries/autocomplete_finder.rb`) would be pretty much the same, except the name changes from `AutocompleteFinderBase` to `AutocompleteFinder`.

The children classes would be like `AutocompleteFinder::Package < AutocompleteFinder`.